recipe/build.sh

```diff
@@ -218,27 +83,25 @@ export TFELHOME=$PREFIX
 export LIBPATH_METIS="${DEST}/metis-${METIS}/lib $PREFIX/lib"
 export INCLUDES_METIS="${DEST}/metis-${METIS}/include $PREFIX/include"
 
-export LIBPATH_MUMPS="${DEST}/mumps-${MUMPS_GPL}/lib $PREFIX/lib"
-export INCLUDES_MUMPS="${DEST}/mumps-${MUMPS_GPL}/include $PREFIX/include ${DEST}/mumps-${MUMPS_GPL}/include_seq"
+export LIBPATH_MUMPS="$PREFIX/lib"
+export INCLUDES_MUMPS="${PREFIX}/include ${PREFIX}/include/mumps_seq"
 
-LDFLAGS="-Wl,--no-as-needed -L${DEST}/med-${MED}/lib -lmed -L${DEST}/hdf5-${HDF5}/lib -lhdf5 -L${DEST}/scotch-${SCOTCH}/lib -lesmumps -lscotch -lscotcherr -lscotcherrexit -lz -ldl -lm ${LDFLAGS}" \
-    FCFLAGS="-fallow-argument-mismatch ${FCFLAGS}" \
-    ./waf_std \
+LDFLAGS="-Wl,--no-as-needed -lscotch -lscotcherr -lscotcherrexit -lz -ldl -lm ${LDFLAGS}" \
+FCFLAGS="-fallow-argument-mismatch ${FCFLAGS}" \
+  ./waf_std \
      --python=$PYTHON \
      --prefix="${PREFIX}" \
      --libdir="${PREFIX}/lib" \
      --install-tests \
      --enable-metis \
      --embed-metis \
      --enable-mumps \
-     --embed-mumps \
      --enable-scotch \
      --enable-mfront \
      --enable-med \
-     --embed-med \
      --enable-hdf5 \
-     --embed-hdf5 \
```

Comment:
I feel like you want to keep these "enables" right?

Reply:
well, if you are referring to the "--embed-*" flags, I believe they look for statically compiled libs. med, hdf5 and mumps I believe are shared libraries?

From the waf config file for med `waftools/med_cfg.py"

```python
group.add_option(
        "--embed-hdf5",
        dest="embed_hdf5",
        default=None,
        action="store_true",
        help="Embed HDF5 libraries as static library",
    )
```

I tried quickly to check what would happen with "--embed-hdf5" and the configuration part failed with:

```
['/home/conda/feedstock_root/build_artifacts/code-aster_1712244556525/_build_env/bin/x86_64-conda-linux-gnu-cc', '-Wl,--export-dynamic', '-Wl,--no-keep-memory', 'test.c.1.o', '-o/home/conda/feedstock_root/build_artifacts/code-aster_1712244556525/work/build/std/.conf_check_928bdad62d1e191bbcbdf50847880f00/testbuild/release/testprog', '-Wl,-Bstatic', '-lhdf5', '-Wl,-Bdynamic', '-lz', '-Wl,--no-as-needed', '-lscotch', '-lscotcherr', '-lscotcherrexit', '-lz', '-ldl', '-lm', '-Wl,-O2', '-Wl,--sort-common', '-Wl,--as-needed', '-Wl,-z,relro', '-Wl,-z,now', '-Wl,--disable-new-dtags', '-Wl,--gc-sections', '-Wl,--allow-shlib-undefined', '-Wl,-rpath,/home/conda/feedstock_root/build_artifacts/code-aster_1712244556525/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib', '-Wl,-rpath-link,/home/conda/feedstock_root/build_artifacts/code-aster_1712244556525/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib', '-L/home/conda/feedstock_root/build_artifacts/code-aster_1712244556525/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib']
err: /home/conda/feedstock_root/build_artifacts/code-aster_1712244556525/_build_env/bin/../lib/gcc/x86_64-conda-linux-gnu/12.3.0/../../../../x86_64-conda-linux-gnu/bin/ld: cannot find -lhdf5: No such file or directory
collect2: error: ld returned 1 exit status
```

Reply:
sorry i misread the  flag. enable vs embed